indico/MaKaC/webinterface/tpls/RoomBookingSearchRoomsResults.tpl

```diff
@@ -90,3 +90,22 @@
   </tr>
 </table>
 <br />
+<script type="text/javascript">
+  $(document).ready(function() {
+      $.fn.extend({
+          tablesorter: $.tablesorter.construct
+      });
+
+      $('.tablesorter').tablesorter({
```

Comment:
that's quite generic. i'd rather give the table an id and select based on that one instead...